app/services/meeting_starter.rb

```diff
@@ -1,17 +1,18 @@
 # frozen_string_literal: true
 
 class MeetingStarter
-  def initialize(room:, logout_url:, presentation_url:, meeting_ended:, recording_ready:)
+  def initialize(room:, logout_url:, presentation_url:, meeting_ended:, recording_ready:, provider:)
```

Comment:
I turned `Metrics/ParameterLists` off for named args.
If you feel that this is too long then I recommend that we change the signature to `initialize(room:, provider: , opts:)` where `opts` (options) is a `Hash(<option_name> => <value>)`.